app/services/labs_channel.rb

```diff
@@ -0,0 +1,16 @@
+require 'httparty'
+
+class LabsChannel
+  def self.post message
+    url = "https://hooks.slack.com/services/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"
```

Comment:
perhaps remove any hardcoding: urls, slack channels ---------> secrets.yml or some other config object, the same url seems to be hardcoded in 3 different places including tests